solr/core/src/java/org/apache/solr/store/blob/util/BlobStoreUtils.java

```diff
@@ -75,6 +76,16 @@ public static String generateMetadataSuffix() {
     return UUID.randomUUID().toString();
   }
 
+  /**
+   * Returns current nano time in millisecond resolution for use in measuring elapsed time.
+   * 
+   * Note: Do not combine this value with currentTimeMillis - currentTimeMillis will return ms relative to epoch
+   * while this method returns ms relative to some arbitrary time
+   */
+  public static long getCurrentNanoTimeInMs() {
```

Comment:
Why not just call it getCurrentTimeMs and let the javadoc make it clear it's not the same as System.currentTimeMillis(), the main reason being it was changed originally to fix ant precommit failures?

Reply:
Sure - I had named this way to avoid combining with currentTimeMillis but if the javadoc is good enough for that I'll rename